pywhat/Data/regex.json

```diff
@@ -239,7 +294,7 @@
     },
     {
        "Name": "Email Address",
-       "Regex": "(^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\\.[a-zA-Z0-9-.]+$)",
+       "Regex": "(?i)(^[a-z0-9_.+-]+@[a-z0-9-]+\\.[a-z0-9-.]+$)",

```

Comment:
Email addresses are super weird, see this regex (maybe make a separate PR for this! This is **huge**!)
https://stackoverflow.com/questions/201323/how-to-validate-an-email-address-using-a-regular-expression
Also make sure that regex isn't super slow :-)